fivetran_provider_async/operators.py

```diff
@@ -55,3 +58,60 @@ def execute_complete(self, context: "Context", event: Optional[Dict[Any, Any]] =
                     event["message"],
                 )
                 return event["return_value"]
+
+    def get_openlineage_facets_on_start(self) -> OperatorLineage:
+        """
+        Default extractor method that OpenLineage will call on execute completion.
+        """
+        # Should likely use the sync hook here to ensure that OpenLineage data is
+        # returned before the timeout.
+        hook = self._get_hook()
+        connector_response = hook.get_connector(self.connector_id)
+        groups_response = hook.get_groups(connector_response["group_id"])
+        destinations_response = hook.get_destinations(connector_response["group_id"])
+        schema_response = hook.get_connector_schemas(self.connector_id)
+        table_response = hook.get_metadata(self.connector_id, "tables")
+        column_response = hook.get_metadata(self.connector_id, "columns")
+
+        for schema in schema_response["schemas"].values():
+
+            inputs = get_dataset(
+                config=connector_response["config"],
+                service=connector_response["service"],
+                table_response=table_response,
+                column_response=column_response,
+                schema=schema,
+                connector_id=self.connector_id,
+                loc="source",
+            )
+
+            outputs = get_dataset(
+                config=destinations_response["config"],
+                service=destinations_response["service"],
+                table_response=table_response,
+                column_response=column_response,
+                schema=schema,
+                connector_id=self.connector_id,
+                loc="destination",
+            )
+
+        job_facets = {
+            "documentation": DocumentationJobFacet(
```

Comment:
Are there other lineage values like `sourceCode` or `ownership` which can be added as per https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#job-facets?